lib/rucio/core/did.py

```diff
@@ -169,7 +169,11 @@ def add_dids(dids, account, session=None):
                 for key in did.get('meta', {}):
                     new_did.update({key: did['meta'][key]})
 
-                new_did.save(session=session, flush=False)
+                new_did.save(session=session, flush=True)
```

Comment:
Please remove these lines for now, as this would have implications on the current production system if deployed this way.